### ISSUE TITLE
Incompatability with loaforcsSoundAPI?

### ISSUE THREAD

**Issue by TheNutFather** [FelixAllard/SCP1507LC#2]
[Error  : Unity Log] NullReferenceException
Stack trace:
loaforcsSoundAPI.Patches.AudioSourcePatch.TryReplaceAudio (UnityEngine.AudioSource __instance, UnityEngine.AudioClip clip, UnityEngine.AudioClip& replacement) (at ./Patches/AudioSourcePatch.cs:85)
loaforcsSoundAPI.Patches.AudioSourcePatch.PlayOneShot (UnityEngine.AudioSource __instance, UnityEngine.AudioClip& clip) (at ./Patches/AudioSourcePatch.cs:40)
(wrapper dynamic-method) UnityEngine.AudioSource.DMD<UnityEngine.AudioSource::PlayOneShot>(UnityEngine.AudioSource,UnityEngine.AudioClip,single)
UnityEngine.AudioSource.PlayOneShot (UnityEngine.AudioClip clip) (at <2b2e47a8d13e478fbaa048194b560eac>:IL_0001)
SCP1507.SCP1507.Scp1507.DoQuackSound () (at ./src/SCP1507/AudioRelay.cs:18)
SCP1507.FlamingoManager.FlamingoManager+<HonkForTimes>d__16.MoveNext () (at ./src/FlamingoManager/FlamingoManager.cs:251)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <e27997765c1848b09d8073e5d642717a>:IL_0026)
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
SCP1507.FlamingoManager.FlamingoManager:HonkManager() (at ./src/FlamingoManager/FlamingoManager.cs:243)

**Comment by FelixAllard** [comment]
I do not think it is an error with LoaforcSoundAPI. That mod patches the UnityAudioSource so it will always be reported when there is an error with the Audio. I will look into what could be causing the issue!
